bench/Cargo.toml

```diff
@@ -12,12 +12,12 @@ build = "build.rs"
 workspace = ".."
 
 [dependencies]
-docopt = "0.8"
+docopt = "1"
 lazy_static = "1"
 libc = "0.2"
 onig = { version = "3", optional = true }
 libpcre-sys = { version = "0.2", optional = true }
-memmap = "0.6"
+memmap = "0.6.2"
```

Comment:
Oh this one is interesting. What requires `0.6.2` specifically?

Reply:
[6.1](https://crates.io/crates/memmap/0.6.1) requires `winapi ^0.2` and `<=0.2.5` do not build on rust 1.0.